src/main/java/com/example/backendspringboottechiteasycontrollerles10/controllers/TelevisionController.java

```diff
@@ -0,0 +1,56 @@
+//package com.example.backendspringboottechiteasycontrollerles10.controllers;
```

Comment:
Waarom is deze klasse uitgecomment? en waarom heet deze klasse "TelevisionController" en niet "TelevisionControllerBonus"?